dask/array/enhanced_numpy.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import, division, print_function
+
+from functools import wraps
+from inspect import getargspec
+
+import numpy as np
+
+from ..utils import ignoring
+
+
+
+# A wrapper for functions that don't provide keepdims to ensure that they do.
+def keepdims_wrapper(a_callable):
```

Comment:
I didn't use `func` as this indicates it is only a function when it could be something else (i.e. method). I would have used `callable`, but this is actually a function in Python. I'm open to other suggestions.